.github/workflows/linux-v2.yml

```diff
@@ -24,6 +24,7 @@ jobs:
 
     - name: Run Linux Unit Tests
       run: sudo go test -covermode=count -coverprofile=profile.cov -tags azurediskv2 ./pkg/azuredisk --temp-use-driver-v2
+      sudo go test -covermode=count -coverprofile=profile.cov -tags azurediskv2 ./pkg/azdiskschedulerextender
```

Comment:
Do we need to run this command with elevated access?

Reply:
Not sure. Since the line above runs the tests with sudo, I did the same with this